go/kbfs/libkbfs/prefetcher_test.go

```diff
@@ -1921,15 +1938,22 @@ func TestPrefetcherReschedules(t *testing.T) {
 	syncBytes, workingBytes = testGetDiskCacheBytes(syncCache, workingCache)
 	setLimiterLimits(limiter, syncBytes, workingBytes)
 
-	t.Log("Release reschedule requests of two more children.")
+	t.Log("Process requests of two more children (aa and ab)")
 	notifySyncCh(t, prefetchSyncCh)
+	waitDoneCh(t, ctx, prefetchDoneCh)
 	notifySyncCh(t, prefetchSyncCh)
+	waitDoneCh(t, ctx, prefetchDoneCh)
 
 	t.Log("Make room in the cache again.")
 	setLimiterLimits(limiter, math.MaxInt64, math.MaxInt64)
 
 	t.Log("Finish all the prefetching.")
 	close(prefetchSyncCh)
+	defer close(prefetchDoneCh)
+	go func() {
+		for range prefetchDoneCh {
+		}
+	}()
```

Comment:
As above, I'm confused why `<-prefetcher.TogglePrefetcher()` isn't good enough...

Reply:
Given the above explanation of needing a second channel, we need to constantly receive on it in a separate goroutine so that the prefetcher can complete it's work.  There's more stuff that needs to be done as we call `waitForPrefetchOrBust`, and it's not important to step through each step with the channels at this point in the test, we've already checked what we care about in the test.